src/main.js

```diff
@@ -1 +1,205 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+
+// Quiz Questions
+// 1. What method would you use to add an element to the end of an array? 
+// array.push method 
+
+// 2. How do you remove the first element from an array? 
+// array.shift 
+
+// 3. What is the difference between splice and slice? 
+// splice can change the array itself, delete or 
+// add DataTransferItemList, slice creates new array using old one
```

Comment:
✅
⚠️ What is `DataTransferItemList`?

Reply:
I was writing texts and then turning them into comments, so it was auto remaking some words when I was doing it like that, so I started turning lines into comments and then writing words.